.pre-commit-config.yaml

```
@@ -24,6 +24,12 @@ repos:
         language: system
         entry: bin/update-from-templates
         pass_filenames: false
+-   repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.11.11
+    hooks:
+        - id: ruff
+          args: [ --fix ]
```

To match the formatting used elsewhere in this file:
```suggestion
          args: [--fix]
```